install_config/cluster_metrics.adoc

```diff
@@ -306,15 +340,15 @@ ifdef::openshift-enterprise[]
 ====
 endif::[]
 
-
-You will need to save your completed file with the file name *_metrics.yaml_*.
+Copy this file to another directory with the file name
+*_metrics-deployer.yaml_*.
```

Comment:
I didn't see a specific reason why we were saying to name the file "metrics.yaml" when the default file is "metrics-deployer.yaml". So I've synced up references and the example commands to just stick with "metrics-deployer.yaml". If there was a reason for renaming it like this, please lmk.